Drop reference to ephemeral storage (#464)
infrawatch/documentation: doc-Service-Telemetry-Framework/modules/con_observability-strategy.adoc

[role="_abstract"]
{Project} ({ProjectShort}) does not include storage backends and alerting tools. {ProjectShort} uses community operators to deploy Prometheus, Alertmanager, Grafana, and Elasticsearch. {ProjectShort} makes requests to these community operators to create instances of each application configured to work with {ProjectShort}.

> Instead of having Service Telemetry Operator create custom resource requests, you can use your own deployments of these applications or other compatible applications, and scrape the metrics Smart Gateways for delivery to your own Prometheus-compatible system for telemetry storage. If you set the observability strategy to use alternative backends instead, persistent storage is not required for {ProjectShort}.

Reviewer: "If you set the observability strategy to use alternative backends instead, persistent storage is not required for STF"

This sentence is bothering me a bit. While it's true that in STF 1.5 you can set `observability_strategy: none` and we won't provision any PVs, in upcoming versions we expect the options to be "none", "use_redhat", "use_community", or "use_hybrid". 

Maybe "If you set the observability strategy not to deploy any backends, persistent storage is not required for STF", which almost seems too obvious to say.

WDYT?

> Instead of having Service Telemetry Operator create custom resource requests, you can use your own deployments of these applications or other compatible applications, and scrape the metrics Smart Gateways for delivery to your own Prometheus-compatible system for telemetry storage. If you set the observability strategy to use alternative backends instead, persistent storage is not required for {ProjectShort}.

Author: ```suggestion
Instead of having Service Telemetry Operator create custom resource requests, you can use your own deployments of these applications or other compatible applications, and scrape the metrics Smart Gateways for delivery to your own Prometheus-compatible system for telemetry storage. If you set the `observabilityStrategy` to `none`, then storage backends will not be deployed so persistent storage will not be required by {ProjectShort}.
```

Author: @csibbitt what do you think about that? I took your thoughts and kinda tried to refactor it a bit.
